libbpf-tools/runqlen.c

```diff
@@ -222,22 +235,22 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
-	obj = runqlen_bpf__open();
-	if (!obj) {
-		fprintf(stderr, "failed to open and/or load BPF object\n");
+	nr_cpus = libbpf_num_possible_cpus();
+	if (nr_cpus < 0) {
+		printf("failed to get # of possible cpus: '%s'!\n",
+		       strerror(-nr_cpus));
 		return 1;
 	}
-
-	nr_cpus = libbpf_num_possible_cpus();
 	if (nr_cpus > MAX_CPU_NR) {
-		fprintf(stderr, "The number of cpu cores is too much, please "
+		fprintf(stderr, "the number of cpu cores is too much, please "
```

Comment:
probably should say "too big", not "too much". "Much" is used for uncountable entities.

Reply:
Fixed, thanks!